packages/data-stores/src/i18n/selectors.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import type { State } from './reducer';
+import type { LocalizedLanguageNames } from './types';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this one necessary?

Reply:
It was left over from an experiment, removing it.